nucleus/__init__.py

```diff
@@ -1111,8 +1110,7 @@ def delete_annotations(
     def append_to_slice(
         self,
         slice_id: str,
-        dataset_item_ids: List[str] = None,
-        reference_ids: List[str] = None,
+        reference_ids: List[str],
```

Comment:
nit: I think you need to update the param doc string to no longer include `dataset_item_ids`